webapp/components/TeaserImage/TeaserImage.vue

```diff
@@ -222,7 +228,7 @@ export default {
 }
 
 .hc-drag-marker-update-post {
-  opacity: 0;
+  opacity: 0.1;
```

Comment:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-01-15T19:28:25Z" title="Wednesday, January 15th 2020, 8:28:25 pm +01:00">Jan 15, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -206,7 +210,7 @@ export default {
 }
 
 .hc-drag-marker-update-post {
-  opacity: 0;
+  opacity: 0.1;
```


i think it's better if you can see a little bit that you can click on it. opacity: 0.1 is almost nothing :) but it becomes clear that there is still something to do.